src/dashboard/src/components/charts/storageTrends/StorageTrendsChart.tsx

```diff
@@ -99,25 +105,18 @@ export const StorageTrendsChart: React.FC<LineChartProps> = ({
       label="Storage Trends"
       loading={loading || !serverHistoryItemsIsReady}
       large={large}
-      showExport
-      exportDisabled
       disclaimer={
         <p className={styles.disclaimer}>*Data shows totals on last available day of each month.</p>
       }
+      showExport={showExport}
+      exportDisabled={exportDisabled}
+      onExport={() => onExport?.(dateRange[0], dateRange[1])}
       filter={
         <div className={styles.date}>
           <DateRangePicker
             values={dateRange}
             onChange={async (values, e) => {
               setDateRange(values);
-              await findServerHistoryItems({
```

Comment:
No need since changing the date range will make this request with the `useEffect`